corehq/apps/domain/views.py

```diff
@@ -564,6 +565,21 @@ def requeue_repeat_record(request, domain):
     return HttpResponse('OK')
 
 
+@require_POST
+@require_can_edit_web_users
+def send_mock_repeater_report(request, domain):
+    try:
+        email_id = request.POST.get('email_id')
+        repeater_id = request.POST.get('repeater_id')
+        data = csv.reader(request.FILES['payload_ids_file'])
+        payload_ids = [row[0] for row in data]
+        send_mock_repeater_payloads(repeater_id, payload_ids, email_id)
+        messages.success(request, _("Successfully emailed payloads."))
+    except Exception as e:
```

Comment:
I think this is overbroad exception catching.  In particular, `send_mock_repeater_payloads` seems like it shouldn't be part of this `try/except` block.

Reply:
Its just that we are accepting a file from the user and it is possible that its not in expected/parsable format or there is some unexpected error. So letting the user know we could not parse the file seems fine here.
Agree on `send_mock_repeater_payloads`.